VideoAPI/Video.API/Controllers/InstantMessageController.cs

```diff
@@ -91,5 +91,30 @@ public async Task<IActionResult> AddInstantMessageToConference(Guid conferenceId
                 return NotFound();
             }
         }
+
+        /// <summary>
+        /// Get list of closed conferences (closed over 30 minutes ago)
+        /// </summary>
+        /// <returns>List of Conference Ids</returns>
+        [HttpGet("expiredIM")]
+        [SwaggerOperation(OperationId = "GetClosedConferences")]
```

Comment:
Change the method and operation name to be reflective of  what is doing. It's getting closed hearings with chat history

Reply:
renamed to GetClosedConferencesWithInstantMessages